Gordon360/Models/ViewModels/RecIM/SportViewModel.cs

```diff
@@ -6,7 +6,7 @@ namespace Gordon360.Models.ViewModels.RecIM
 {
     public class SportViewModel
     {
-        public int ID { get; set; }
+        public int? ID { get; set; }
```

Comment:
Should you have a separate `SportUploadViewModel` where the ID is not required? I can't think of why (other than an upload/update) a sport id would need to be null.